cvFit/src/app/app.module.ts

```diff
@@ -19,35 +19,62 @@ import { ExerciseListComponent } from './exercises/exercise-list.component';
 import { ExerciseDetailComponent } from './exercises/exercise-detail.component';
 import { WelcomeComponent } from './home/welcome.component';
 import { ContactUsComponent } from './form/contactUs.component';
-import { LoginComponent } from './form/login.component';
-import { ProfileComponent } from './form/profile.component';
 import { BlogComponent } from './blog/blog.component';
 import { WorkoutListComponent } from './exercises/workoutList/workoutList.component';
 import { PostComponent } from './blog/post/post.component';
 import { DiscussionComponent } from './blog/discussion/discussion.component';
-
-import { AuthService } from './form/auth.service';
 import { CalculatorComponent } from './macroCalculator/calculator.component';
 import { MacroTableComponent } from './macroCalculator/macro-table.component';
 import { CalcService } from './macroCalculator/calculator.service';
 
+import { MsalInterceptor, MsalModule, MsalRedirectComponent, MsalGuard } from '@azure/msal-angular';
+import { InteractionType, PublicClientApplication } from '@azure/msal-browser';
+import { AppRoutingModule } from './app-routing.module';
+import { AZUREprofileComponent } from './azureprofile/azureprofile.component';
+import { HomeComponent } from './home/home.component';
+
+const isIE = window.navigator.userAgent.indexOf('MSIE ') > -1 || window.navigator.userAgent.indexOf('Trident/') > -1;
+
 @NgModule({
   declarations: [
     AppComponent,
     ExerciseListComponent,
     ExerciseDetailComponent,
     ContactUsComponent,
     WelcomeComponent,
-    LoginComponent,
-    ProfileComponent,
     BlogComponent,
     WorkoutListComponent,
     PostComponent,
     DiscussionComponent,
     CalculatorComponent,
-    MacroTableComponent
+    MacroTableComponent,
+    AZUREprofileComponent,
+    HomeComponent
   ],
   imports: [
+    AppRoutingModule,
+    MsalModule.forRoot( new PublicClientApplication({
+      auth: {
+        clientId: '6063621e-ae81-4c3e-b0ee-6c9486c01725', // Application (client) ID from the app registration
```

Comment:
this should be pulled from environment or some other place. you could do this `clientId: environment.msal.clientId` and create a structure for that in your environment file. This better aligns with eventually injecting that clientId in from a secret vault or library.

Reply:
I would do this for the authority as well.